_posts/2021-05-14-papers-teaching.md

```diff
@@ -0,0 +1,27 @@
+---
+site: freiburg
+title: 'Two New Teaching Papers using Galaxy'
+tags:
+- paper
+- training
+- TIaaS
+---
+
+<br>
+Two hand-in-hand teaching papers have been published this week using Galaxy as a teaching platform. The common frame encompassing both is the effects of the lockdown during the COVID-19 pandemic. University lessons and workshop have been forced to move to an online setting. 
+
+The [first manuscript](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008922), a review, sets the ground for bioinformatics teaching during the lockdown, proposing constructivism as an appropriate methodology for students to learn in an online context.
+<br>
+
+{% twitter https://twitter.com/gxytraining/status/1393101522704101377 %}
+
+<br>
+The [second manuscript](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008923#sec003) describes how Galaxy can facilitate online teaching using the Galaxy Training Network materials, Training Infrastructure as a Service and many features such as data, history and workflow sharing, with a concrete proposed format to make the most of your training events.
+
+<br>
+
+{% twitter https://twitter.com/gxytraining/status/1393104039236317188 %}
+
+<br>
+
+Read them both to get some practical tips for your next training!
```

Comment:
Do you want to mentione that we already used those recommendations in 3 highly sucessful workshops?

Reply:
done too